pydra/engine/workers.py

```diff
@@ -66,22 +75,42 @@ def close(self):
 
 
 class ConcurrentFuturesWorker(Worker):
-    def __init__(self, nr_proc=4):
-        self.nr_proc = nr_proc
+    def __init__(self, nr_proc=2, loop=None):
+        super(ConcurrentFuturesWorker, self).__init__()
+        self.nr_proc = nr_proc or mp.cpu_count()
```

Comment:
this change is not needed, `nr_proc` could be `None`. I was planning to change it to `None` in `__init__` as default.

Reply:
yep, this was my testing, i'll just set `nr_proc`'s default to `None`. (though we should provide some api for the user to alter this)